jump-core/src/main/java/com/bitdecay/jump/level/builder/LevelBuilder.java

```diff
@@ -274,11 +284,11 @@ public void updateOwnNeighbors(int x, int y) {
 		}
 		// check left
 		if (ArrayUtilities.onGrid(grid, x - 1, y) && grid[x - 1][y] != null) {
-			if (grid[x-1][y].oneway) {
-				grid[x][y].collideNValue &= Direction.NOT_LEFT;
-			} else {
+			//if (grid[x-1][y].oneway) {
+			//	grid[x][y].collideNValue &= Direction.NOT_LEFT;
+			//} else {
```

Comment:
Ah indeed. Forgot to uncomment them.
On Dec 17, 2015 11:22 PM, "Logan" <notifications@github.com> wrote:

> In
> jump-core/src/main/java/com/bitdecay/jump/level/builder/LevelBuilder.java
> <https://github.com/bitDecayGames/Jump/pull/161#discussion_r47997642>:
>
> > @@ -274,11 +284,11 @@ public void updateOwnNeighbors(int x, int y) {
> >  		}
> >  		// check left
> >  		if (ArrayUtilities.onGrid(grid, x - 1, y) && grid[x - 1][y] != null) {
> > -			if (grid[x-1][y].oneway) {
> > -				grid[x][y].collideNValue &= Direction.NOT_LEFT;
> > -			} else {
> > +			//if (grid[x-1][y].oneway) {
> > +			//	grid[x][y].collideNValue &= Direction.NOT_LEFT;
> > +			//} else {
>
> I think we still want these. Commenting these out will result in bugs when
> one-way platforms butt up against regular terrain.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitDecayGames/Jump/pull/161/files#r47997642>.
>